src/core/json-schema-components.js

```diff
@@ -87,19 +86,18 @@ export class JsonSchema_string extends Component {
                      className={ errors.length ? "invalid" : ""}
                      title={ errors.length ? errors : ""}
                      onChange={ this.onChange }
-                     disabled={isDisabled}/>)
+                     disabled={ isDisabled }/>)
     }
     else {
-      return (<DebounceInput
+      return (<Input
                      type={ schema.format === "password" ? "password" : "text" }
                      className={ errors.length ? "invalid" : ""}
                      title={ errors.length ? errors : ""}
-                     value={value}
-                     minLength={0}
-                     debounceTimeout={350}
-                     placeholder={description}
+                     value={ value }
+                     minLength={ 0 }
+                     placeholder={ description }
                      onChange={ this.onChange }
```

Comment:
If we need the same behaviour, we could change either this line to onChange={`_.debounce(this.onChange, 350)`} or onChange to `onChange = _.debounce(() => this.props.onChange(this.state.value))`.

Reply:
I'm pretty sure we do need the behavior - I'll need to look back into some old tickets to confirm.